skype_send_message.py

```diff
@@ -1,29 +1,26 @@
+import keyring
 import argparse
-import bcrypt
 from skpy import Skype
-import time
 parser = argparse.ArgumentParser()
 parent_parser = argparse.ArgumentParser(add_help=False)
+parent_parser.add_argument('--servicename', action="store")
```

Comment:
@idumenskyi could you please elaborate on this? Do we need this?

Reply:
@ekondrashev sorry, and in right really this parser is not needed, I used it because thought that as  argument of "keyring" him need parsing.